include/Utils/DIPUtils.h

```diff
@@ -24,6 +24,21 @@
 
 #include "Utils/Utils.h"
 
+Value insertFMAOp(OpBuilder &builder, Location loc, VectorType type,
```

Comment:
Please add a comment line explaining the motive of this function, similar to other functions in `DIPUtils.h`.

Reply:
Added.